source/Calamari.Common/Plumbing/Deployment/PackageRetention/IUsageDetails.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Calamari.Common.Plumbing.Deployment.PackageRetention
+{
+    public interface IUsageDetails
+    {
+        CacheAge CacheAge { get; }
+        DateTime DateTime { get; }
```

Comment:
What's the DateTime property used for?

Reply:
It will be used for expiring stale locks, and _may_ be used with the caching algorithm.